README.md

```diff
@@ -1253,6 +1253,8 @@ The use of Amazon Voice Focus and background blur via this SDK involves the down
 
 The use of Amazon Voice Focus and background blur runtime code is subject to additional notices. See [this Amazon Voice Focus NOTICES file](https://static.sdkassets.chime.aws/workers/NOTICES.txt) and [background blur NOTICES file](https://static.sdkassets.chime.aws/bgblur/workers/NOTICES.txt) for details. You agree to make these additional notices available to all end users who use Amazon Voice Focus and background blur runtime code via this SDK.
 
+You and your End Users are responsible for all Content (including any images) uploaded for use with video background replacement, and must ensure that such Content does not violate the law, infringe or misappropriate the rights of any third party, or otherwise violate a material term of your agreement with Amazon (including the documentation, the AWS Service Terms, or the Acceptable Use Policy).
```

Comment:
Do we need to capitalize End Users and Content?

Reply:
This was given to us by the legal team